src/Finbuckle.MultiTenant.EntityFrameworkCore/Stores/EFCoreStore/EFCoreStoreDbContext.cs

```diff
@@ -13,7 +13,7 @@ public EFCoreStoreDbContext(DbContextOptions options) : base(options)
         {
         }
 
-        public DbSet<TTenantInfo> TenantInfo { get; set; }
+        public DbSet<TTenantInfo> TenantInfo => Set<TTenantInfo>();
```

Comment:
This is the one file I wasn't sure how to handle. Let me know what you think of this change and whether it will break anyone.